### ISSUE TITLE
SC triggered to download completed stuff

### ISSUE THREAD

**Issue by loungebob** [SickChill/SickChill#6492]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: current
OS: win10
Browser: FF current
What you did: nothing
What happened: SC downloaded stuff
What you expected: for SC to not download stuff
Logs:
```
PASTE LOGS HERE
```
for the 3 time in the last couple of days SC downloaded TB of data of already completed shows. something triggers SC to think that stuff is missing. how do I prevent this?

**Comment by miigotu** [comment]
Is it completed and showing green (preferred) on the show list or yellow (allowed)?

**Comment by loungebob** [comment]
it’s green. archived. but I manually chose that to stop SC from downloading stuff. I did this for about 20 shows. and then for about 30 more a couple of days ago. if you’re asking if SC replaces the initially downloaded stuff with better copies, that is not the case. it downloads the exact same content again. I think something outside SC triggers this.

**Comment by miigotu** [comment]
Maybe when tvdb was down? If you have a metered connection SC that sucks.

**Comment by loungebob** [comment]
yeah yeah I get, that it is something like tvdb’s suckybuggy api. or one of the SC internal automatic processes that trigger this. and I dont really care about the bandwidth, it’s peanuts. however having the drives filled and post-processing tools working on useless/redundant stuff is annoying and using resources unneccessarily. 
cant I tell SC to never do that (mass download) from now on until I manually do something? I mean, normal operations continue (daily/weekly eps) but anything older than a certain date wont be done until I manually approve?

**Comment by miigotu** [comment]
Maybe it's nzbtomedia/torrenttomedia triggering it?

**Comment by loungebob** [comment]
it just did it again for all Hawaii 5.0 and all Empire 2015 episodes. I dont use any of those scripts. I use SC only to download shows. No post processing is done. download to blackhole, everything else from there is done by other tools.

**Comment by miigotu** [comment]
IDK how you are postprocessing and marking episodes from snatched to downloaded without SC postprocessing or nzbtomedia, but it sure seems like something is screwing up there. Mine has never redownloaded something that was complete except once when there was a piece of code that deleted episodes if tvdb didn't respond. That code is fixed, and haven't had the problem since.

**Comment by loungebob** [comment]
there is a good chance I may not have explained the roles properly. I add all shows I want in SC. SC downloads the nzb/torrent to a blackhole directory. qbittorrent/sab/nzbget pick up from the blackhole and download. 
I assumed after SC downloaded to the blackhole its job is done and it marks the download as successful and done, no? (unless there is a proper/repack/better version).
In this scenario I cant see a reason for SC to redownload complete shows. 
anyway, I also assumed others have the same issue with SC but since that does not seem to be the case, it’ll be something in my installation and I’ll have a look at it later.

**Comment by miigotu** [comment]
No, once it is downloaded to black hole SC is not done and nothing is marked done until the download completes in your download client and post processes the download.

**Comment by nydave69** [comment]
I've had the same thing happen the other day..

I had Cheers TV show downloaded years ago at 1080p, it was archived, paused as complete
For some reason the other day I see Cheers listed as downloaded (SC doesn't remove them from the list good thing)
Sure enough it replaced all of my 1080p episodes with 480p..
Good thing the recycle bin wasn't emptied, was able to move them back. fixed the damage.
Rescan worked getting back in the database.

Today I found an odd thing...
Avenue 5 season one was completed months ago.  All 1080p..

Avenue 5 (2020) (S01E05) (2020-02-16) (1080p-h264 EAC3-6ch) He's Only There to Stop His Skeleton Falling Over.mkv

from season 1 was replaced in the data base with.
tvshow-trailer.mp4

That trailer is in the show folder not in the season 1 folder,  A rescan wont fix it for some reason.
The real episode is in the Season 1 folder.

It's only wrong in the SC database, Need to know how to fix episode 5 in the database.

I'm also looking to see if any others are like this.
